tests/FSharp.Test.Utilities/CompilerAssert.fs

```diff
@@ -151,19 +151,22 @@ let main argv = 0"""
                 pInfo.UseShellExecute <- false
 
                 let p = Process.Start(pInfo)
-                let succeeded = p.WaitForExit(10000)
+                let timeout = 30000
```

Comment:
For some reason, it was taking a bit more time to get all dependencies sometimes. So, increased default timeout.